05LessonFive/Checkpoint1/css/style.css

```diff
@@ -21,14 +30,15 @@
   padding-left: 5px;
   padding-right: 5px;
   padding-bottom: 5px;
```

Comment:
Padding can be done all on one line if all sides have the same padding. 
Ex: 
`padding: 5px;`